data_loader.py

```diff
@@ -191,7 +191,36 @@ def check_files(self, file_list):
                     return False
             return True
 
-    def validate_files(self, cheat_mode, file_list, max_violations, temp_folder, verbose):
+    def validate_delete_files(self, file_list):
+        validation_result = True
+        with self.driver.session() as session:
+            for txt in file_list:
+                file_encoding = check_encoding(txt)
+                with open(txt, encoding=file_encoding) as in_file:
+                    reader = csv.DictReader(in_file, delimiter='\t')
+                    for org_obj in reader:
+                        obj = self.cleanup_node(org_obj)
+                        id_field = self.schema.get_id_field(obj)
+                        if id_field not in obj.keys():
+                            self.log.error(f'Required id field {id_field} is missing, validation failed')
+                            return False
+                        elif obj[id_field] is None:
```

Comment:
Will it handle empty string in id_field?

Reply:
the script will log the error Invalid data at line 2: "Required property: "file_id" is empty!"! and the error will also be in the data validation result file